docs/Dashactyl/installation.md

```diff
@@ -27,12 +27,14 @@ sudo apt update && sudo apt upgrade
 # installing git CLI
 sudo apt install git
 
-# installing NPM
-sudo apt install npm
-
-# installing NodeJS
+# installing NodeJS & NPM on Ubuntu
 curl -fsSL https://deb.nodesource.com/setup_14.x | sudo bash -
-sudo apt install nodejs
+sudo apt install -y nodejs
+
+# installing NodeJS & NPM on Debian, as root
+curl -fsSL https://deb.nodesource.com/setup_14.x | bash -
```

Comment:
Why not add sudo here? Would work perfectly fine, it's the same commands for Ubuntu/Debian.

Reply:
Some Debian version doesn't have sudo installed by default, which is easily solvable by doing `apt install sudo`.
what do you think? should I remove the Debian part

Reply:
Oh, yeah, that's true, my bad.